doc/user/logging.md

```diff
@@ -2,29 +2,48 @@
 
 Operator SDK-generated operators use the [`logr`][godoc_logr] interface to log. This log interface has several backends such as [`zap`][repo_zapr], which the SDK uses in generated code by default. [`logr.Logger`][godoc_logr_logger] exposes [structured logging][site_struct_logging] methods that help create machine-readable logs and adding a wealth of information to log records.
 
-## Setting the logger
+## A simple example
 
-Operators set the logger for all operator logging in [`cmd/manager/main.go`][code_set_logger]:
+Operators set the logger for all operator logging in [`cmd/manager/main.go`][code_set_logger]. To illustrate how this works, try out this simple example:
 
 ```Go
+package main
+
 import (
 	"github.com/operator-framework/operator-sdk/pkg/log/zap"
 	"github.com/spf13/pflag"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
-func main() {
-  pflag.CommandLine.AddFlagSet(zap.FlagSet())
-  pflag.Parse()
-  logf.SetLogger(zap.Logger())
-  log := logf.Log.WithName("cmd")
+var globalLog = logf.Log.WithName("global")
 
-  ...
+func main() {
+	pflag.CommandLine.AddFlagSet(zap.FlagSet())
+	pflag.Parse()
```

Comment:
Formatting.

Reply:
Disregard. Looks fine in rendered file.

Reply:
This is just me switching everything from spaces to tabs (since that's what go fmt uses).